### PR TITLE
Added a connector for portuguese radio Oxigenio FM

### PR DESCRIPTION
**Pull request by fckawe**
**Describe the changes you made**
Added a connector for portuguese radio Oxigenio FM.

**Additional context**
The connector was requested and discussed with issue #2835.

### CODE REVIEW

File: src/connectors/oxigenio.fm.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Connector.artistTrackSelector = 'div[id^="wp"][id$="texttitle"] span:nth-child(1)';
+
+Connector.playButtonSelector = 'div[id^="wp"][id$="buttonplay"]';
+
+Connector.isStateChangeAllowed = () => {
+	const artistTrack = Util.getTextFromSelectors(Connector.artistTrackSelector);
+	return artistTrack.includes('-') && !artistTrack.match(/^\s*OXIGENIO - /);
+};
+
+Connector.onReady = Connector.onStateChanged;
```
Comment (inverse) [inline src/connectors/oxigenio.fm.js:14]: I think we can drop this one

Comment (fckawe) [reply src/connectors/oxigenio.fm.js:14]: Sorry, could you explain in detail what you mean?

Comment (inverse) [reply src/connectors/oxigenio.fm.js:14]: My understanding of this onReady behaviour is when you need to override the default behaviour when the scripts  are injected.

See: https://github.com/inverse/web-scrobbler/blob/882fdba3702795d0a337a27323523cd81c147263/src/core/content/connector.js#L418

by default the state transitions are handled via the `playerSelector` which is watched for mutations.

Comment (fckawe) [reply src/connectors/oxigenio.fm.js:14]: Okay, thanks for this explanation! I wasn't sure what's that line for, tbh.
I'll remove that, check if it works without it and if so, commit.

Comment (inverse) [reply src/connectors/oxigenio.fm.js:14]: Yeah I think some other connectors also used this pattern when in reality they don't need to. Aside that the additions looks good. Thanks for your support 🙏🏼

**Review by inverse — APPROVED** [review]
Thanks!

**Pull request merged**